adcom1/category_taxonomy.go

```diff
@@ -7,10 +7,11 @@ type CategoryTaxonomy int64
 //
 // Values of 500+ hold vendor-specific codes.
 const (
-	CatTaxIABContent10  CategoryTaxonomy = 1 // 1	IAB Content Category Taxonomy 1.0.
-	CatTaxIABContent20  CategoryTaxonomy = 2 // 2	IAB Content Category Taxonomy 2.0: www.iab.com/guidelines/taxonomy
-	CatTaxIABProduct10  CategoryTaxonomy = 3 // 3	IAB Ad Product Taxonomy 1.0.
-	CatTaxIABAudience11 CategoryTaxonomy = 4 // 4	IAB Audience Taxonomy 1.1.
-	CatTaxIABContent21  CategoryTaxonomy = 5 // 5	IAB Content Category Taxonomy 2.1.
-	CatTaxIABContent22  CategoryTaxonomy = 6 // 6	IAB Content Category Taxonomy 2.2
+	CatTaxIABContent10  CategoryTaxonomy = 1 // IAB Content Category Taxonomy 1.0.
+	CatTaxIABContent20  CategoryTaxonomy = 2 // IAB Content Category Taxonomy 2.0: www.iab.com/guidelines/taxonomy
```

Comment:
Do the comments next to these taxonomies need to updated to include that these two are deprecated/not to recommend for use? I noticed both taxonomies had definitions referencing being deprecated/not recommended in the AdCOM spec.

Reply:
Those were deprecated in a previous release. Must've missed it. Good catch. Updated enumeration comments to match the spec.